src/main/java/org/jabref/logic/texparser/TexBibEntriesResolver.java

```diff
@@ -9,25 +10,29 @@
 
 import org.jabref.logic.importer.ImportFormatPreferences;
 import org.jabref.logic.importer.OpenDatabase;
-import org.jabref.logic.preferences.TimestampPreferences;
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.logic.l10n.Localization;
 import org.jabref.model.database.BibDatabase;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.texparser.Citation;
 import org.jabref.model.texparser.LatexBibEntriesResolverResult;
 import org.jabref.model.texparser.LatexParserResult;
 import org.jabref.model.util.FileUpdateMonitor;
 
+import org.mariadb.jdbc.internal.logging.Logger;
```

Comment:
Wrong logger

Reply:
Huh? 🤣

Reply:
IntelliJ with Alt+Return

Reply:
No copy paste without using your brain, or in this case do not trust the idea without thinking